Makefile

```diff
@@ -1,8 +1,16 @@
 EXAMPLE_SRCS=$(shell find examples -name '*.idr')
 EXAMPLE_TARGETS=$(EXAMPLE_SRCS:examples/%.idr=examples/%.vim)
+EXAMPLE_IBCS=$(EXAMPLE_SRCS:examples/%.idr=examples/%.ibc)
 
 .PHONY: examples
+
 examples: $(EXAMPLE_TARGETS)
 
-examples/%.vim: examples/%.idr
-	idris $< -p vimscript --codegen vim -o $@
+examples/%.vim: examples/%.idr | ipkg
+	idris $^ -p vimscript --codegen vim -o $@
+
+ipkg: 
```

Comment:
`ipkg` should have a `PHONY` entry, no?

Reply:
I thought so too, but I think the Make manual says no phony entry should be a prerequisite of a real file.

Reply:
@mrkgnao That's true. This is probably more a symptom of the examples depending on something outside the build (the installed ipkg). Would it make sense to have `ipkg` depend on all Idris source files and the package file?

Reply:
Yeah, that makes sense.